### ISSUE TITLE
This is not working  i only got student informtion

### ISSUE THREAD

**Issue by shubhamJ12** [virresh/ShortCuts#1]

**Comment by virresh** [comment]
The app was never constructed or status of Android Developer Challenge updated because I never heard back from the folks organizing challenge

Basically I didn't have enough expertise in the ML on Android part of app and was looking for guidance under the Android Development Challenge
This repo only has a POC written in python


I'm not sure what you mean by it's not working?

Did you even try the code source files?

**Comment by shubhamJ12** [comment]
yes,Will this code help us to summarise the video by extracting speech from it. I am getting the output as shortened no of frames and total frames amd am unable to see the cut video and summary
Can u help me on this?

**Comment by virresh** [comment]
You misunderstood the project then.

It's not trying to extract speech and summarise audio

The output with shortened frames is the goal here. Capturing audio isn't an
immediate goal.

**Comment by shubhamJ12** [comment]
how can i see the final cut video?
